core/vm/interpreter_zkevm.go

```diff
@@ -1,6 +1,28 @@
 package vm
 
-import "github.com/ledgerwatch/log/v3"
+import (
+	"github.com/ledgerwatch/erigon/zk/sequencer"
+	"github.com/ledgerwatch/log/v3"
+)
+
+// Config are the configuration options for the Interpreter
+type Config struct {
```

Comment:
maybe we can have a wrapper struct `ZkConfig { config: Config collector: *CounterCollector }` ?

Reply:
Good idea, saves some confusion, made the change, will push shortly